src/component/contents/DraftEditorLeaf.react.js

```diff
@@ -161,11 +167,20 @@ class DraftEditorLeaf extends React.Component<Props> {
       styleObj = Object.assign(styleObj, newStyles);
     }
 
+    // skip inserting an empty `class` property to the element
+    const classes = classNames.length
+      ? // we don't use `cx` here to allow users to take care their classNames
+        // minification by themselves. This is handy when used libraries like `jss`
+        // or similar
+        {className: classNames.join(' ').trim()}
+      : {};
```

Comment:
Setting classes to null or undefined would already create a dom node without an empty class attribute.

https://codesandbox.io/s/upbeat-yalow-uwkoh

Spreading here is just unnecessary and not good for perf.

Reply:
thanks for looking into this

Reply:
changed